packages/autocomplete-core/src/types/AutocompleteSource.ts

```diff
@@ -58,6 +58,10 @@ export interface AutocompleteSource<TItem extends BaseItem> {
    * You can trigger different behaviors based on the event `type`.
    */
   onActive?(params: OnHighlightParams<TItem>): void;
+  /**
+   * Applied to data-autocomplete-source-id on the section source container
```

Comment:
```suggestion
   * Identifier for the source.
```

In the JS package, we can add a reference to `data-autocomplete-source-id`, but it doesn't exist in the core package.

Example:

> Identifier for the source.
> It is used as value for the `data-autocomplete-source-id` attribute of the source `section` container.

Reply:
Make sense! 

Where should I add this reference? In the JS package, `AutocompleteSource` is an intersection of the core `AutocompleteSource` so there's no references yet

Reply:
I know that this works but I'm not sure if we can do better (cc @Haroenv):

```ts
interface AutocompleteCoreSourceWithDocs<TItem extends BaseItem>
  extends AutocompleteCoreSource<TItem> {
  /**
   * Identifier for the source.
   * It is used as value for the `data-autocomplete-source-id` attribute of the source `section` container.
   */
  sourceId: string;
}

export type AutocompleteSource<TItem extends BaseItem> = WithTemplates<
  AutocompleteCoreSourceWithDocs<TItem>,
  TItem
>;
```

Reply:
Added in https://github.com/algolia/autocomplete.js/pull/429/commits/60f61c2e6b04e0282ab9da915419f17177b6ac07

Reply:
I was confused first since I thought "docs" was talking about "documents", not "documentation", but it does make sense like this. I've looked around and can't find an alternative